### PR TITLE
feat: add doc on how to use different connection modes

### PR DESCRIPTION
**Pull request by mdmohsin7**
This PR adds a `How to Use` section in the `README.md` file. This is being done to make it easy for others to understand how they are supposed use the three different types of connections. Below is the content that has been added.

### CLN gRPC

- Launch the app
- In connection type dropdown, select `ClientMode.grpc`
- Provide the directory path consisting of the TLS Certificate\
  Example: `/ssd/home/usr/lnd`. The lnd folder consists of the TLS Cert file
- Enter the host alongside the port of your lightning node. You can find it by running `lightning-cli getinfo`\
  Example: `120.154.200.80:19735`

### LN Lambda

- Launch the app
- In connection type dropdown, select `ClientMode.lnlambda`
- Type in the node id and host (alongside the port). You can find both the values by running `lightning-cli getinfo`
- For the Lambda Server field, type in `https://lnlambda.lnmetrics.info`
- Type in the rune. If you don't exactly know what a rune is, you can generate one by running `lightning-cli createrune`. If you have already generated a rune in the past and want to use that only, then you can find the past runes by running `lightning-cli showrunes`.

### CLN RPC

- Launch the app
- In connection type dropdown, select `ClientMode.unixSocket`
- Provide the directory path consisting of the lightning-rpc file. You will mostly find it in the .lightning folder\
  Example: `/home/usr/xyz/.lightning/`

### CODE REVIEW

**Comment by vincenzopalazzo** [comment]
CI is failing, but please add some description on your PR a empty body is really bad

**Comment by mdmohsin7** [comment]
> CI is failing, but please add some description on your PR a empty body is really bad

Sorry for the empty PR. I have added the description now.

**Comment by mdmohsin7** [comment]
Hi @vincenzopalazzo, the CI is failing because of the following issues returned by flutter analyse:
```
info • 'textScaleFactor' is deprecated and shouldn't be used. Use textScaler instead. Use of textScaleFactor was deprecated in preparation for the upcoming nonlinear text scaling support. This feature was deprecated after v3.12.0-2.0.pre • lib/components/bottomsheet.dart:60:25 • deprecated_member_use
info • Don't use 'BuildContext's across async gaps • lib/views/pay/numberpad_view.dart:35:7 • use_build_context_synchronously
info • Don't use 'BuildContext's across async gaps • lib/views/pay/numberpad_view.dart:46:7 • use_build_context_synchronously
info • Don't use 'BuildContext's across async gaps • lib/views/pay/pay_view.dart:40:7 • use_build_context_synchronously
info • Don't use 'BuildContext's across async gaps • lib/views/request/request_view.dart:42:7 • use_build_context_synchronously
info • Don't use 'BuildContext's across async gaps • lib/views/request/request_view.dart:60:7 • use_build_context_synchronously
info • Don't use 'BuildContext's across async gaps • lib/views/request/request_view.dart:73:7 • use_build_context_synchronously
info • 'textScaleFactor' is deprecated and shouldn't be used. Use textScaler instead. Use of textScaleFactor was deprecated in preparation for the upcoming nonlinear text scaling support. This feature was deprecated after v3.12.0-2.0.pre • lib/views/request/request_view.dart:360:[19](https://github.com/dart-lightning/lndart.clnapp/actions/runs/7798729177/job/21268017041?pr=139#step:7:20) • deprecated_member_use
```
Should I fix them and push in this PR itself or should I open a separate PR for them?

**Comment by vincenzopalazzo** [comment]
>Should I fix them and push in this PR itself or should I open a separate PR for them?

Better a separate PR

File: README.md [hunk 1]
```diff
@@ -117,6 +121,33 @@ flutter build ipa
...
 - The Xcode build archive (.xcarchive file) generated can be found inside `[project]/build/ios/archive/` directory and an App Store app bundle (.ipa file) in `[project]/build/ios/ipa/`
 
+## How to Use
+
+### CLN gRPC
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.grpc`
+- Provide the directory path consisting of the TLS Certificate\
+  Example: `/ssd/home/usr/lnd`. The lnd folder consists of the TLS Cert file
```
Comment (vincenzopalazzo) [inline README.md:131]: did you use chat gpt to generate this? we do not support `lnd`

File: README.md [hunk 2]
```diff
@@ -117,6 +121,33 @@ flutter build ipa
...
+- Type in the node id and host (alongside the port). You can find both the values by running `lightning-cli getinfo`
+- For the Lambda Server field, type in `https://lnlambda.lnmetrics.info`
+- Type in the rune. If you don't exactly know what a rune is, you can generate one by running `lightning-cli createrune`. If you have already generated a rune in the past and want to use that only, then you can find the past runes by running `lightning-cli showrunes`.
+
+### CLN RPC
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.unixSocket`
+- Provide the directory path consisting of the lightning-rpc file. You will mostly find it in the .lightning folder\
+  Example: `/home/usr/xyz/.lightning/`
```
Comment (vincenzopalazzo) [inline README.md:149]:
```suggestion
  Example: `/home/usr/xyz/.lightning/<network>`
```

File: README.md [hunk 3]
```diff
@@ -117,6 +121,33 @@ flutter build ipa
...
+- Enter the host alongside the port of your lightning node. You can find it by running `lightning-cli getinfo`\
+  Example: `120.154.200.80:19735`
+
+
+### LN Lambda
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.lnlambda`
+- Type in the node id and host (alongside the port). You can find both the values by running `lightning-cli getinfo`
+- For the Lambda Server field, type in `https://lnlambda.lnmetrics.info`
```
Comment (vincenzopalazzo) [inline README.md:141]: Maybe you should make also another Pr to make the lnlambda.lnmetrics.info a default option?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mdmohsin7) [reply README.md:131]: No I didn't. I guess I did a mistake while checking the directory. It has to be `/home/usr/xyz/.lightning/bitcoin` (incase of mainnet)  and `/home/usr/xyz/.lightning/testnet` (incase of testnet). Am I right?

For my node, it is `/home/usr/xyz/.lightning/testnet`

File: README.md (hunk 3 above)
Comment (mdmohsin7) [reply README.md:141]: Yes can do. Should I do that after this PR is merged?

File: README.md (hunk 1 above)
Comment (vincenzopalazzo) [reply README.md:131]: yes you are right

File: README.md (hunk 3 above)
Comment (vincenzopalazzo) [reply README.md:141]: yes another PR sounds good
